newswires/client/src/sharedTypes.ts

```diff
@@ -22,6 +22,7 @@ const FingerpostContentSchema = z
 		language: z.string(),
 		usage: z.string(),
 		location: z.string(),
+		abstract: z.string().optional(),
```

Comment:
```suggestion
		abstract: z.string(),
```
this typedef is `partial`, so the optional here is redundant, all fields on this object are optional